targets.py

```diff
@@ -13,3 +15,25 @@ def before_restore(self):
         if not isdir(self.path):
             print("Creating empty directory " + self.path)
             makedirs(self.path)
+
+
+class NamedVolumeTarget:
+    def __init__(self, name):
+        self.name = name
+
+    @property
+    def id(self):
+        return "Named volume: " + self.name
+
+    def _volume_exists(self):
+        text = run(
+            ["docker", "volume", "ls", "-q"],
+            stdout=PIPE, universal_newlines=True
+        ).stdout
+        names = text.split('\n')
+        return self.name in names
+
+    def before_restore(self):
```

Comment:
Is this ever invoked?

Reply:
No, think this was just copied over from previous backup script, but we can delete it for now until we come to write the restore part of bb8.